js/panel_controller.js

```diff
@@ -19,11 +19,19 @@
 
     window.drawAttention = function() {
         if (inboxOpened && !inboxFocused) {
+            if( window.keepClear ) {
+                clearInterval( window.keepClear );
+                delete window.keepClear;
+            }
             extension.windows.drawAttention(inboxWindowId);
         }
     };
     window.clearAttention = function() {
-        extension.windows.clearAttention(inboxWindowId);
+        if( window.keepClear ) {
+            clearInterval( window.keepClear );
+            delete window.keepClear;
+        }
+        window.keepClear = setInterval( () => extension.windows.clearAttention(inboxWindowId), 100 );
```

Comment:
Also, is there any way we could increase that `setInterval` timing? Every 100ms is 10 times a second, and that seems like too much. Maybe once every five seconds would still work? It would certainly result in less of an impact to the performance of the app.